outernet/federation/v1alpha/federation.proto

```diff
@@ -261,7 +261,7 @@ message InterconnectionPointFilters {
 // Requests will be treated as bidirectional by default, unless otherwise
 // specified in the proto documentation.
 enum Directionality {
```

Comment:
I would change to:

```
enum Directionality {
  DIRECTIONALITY_UNSPECIFIED = 0;
  DIRECTIONALITY_BIDIRECTIONAL = 1;
  DIRECTIONALITY_X_TO_Y = 2;
  DIRECTIONALITY_Y_TO_X = 3;
}
```

Reply:
(To clarify: Note that the MR as proposed discards BIDIRECTIONAL ability to specify)